### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by Chike1990**
In this milestone, I set up the UI of the calculator app

### CODE REVIEW

File: math/src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable  */
+import React, { Component } from 'react';
+
+import "./Calculator.css";
+
+class Calculator extends React.Component {
```
Comment (smcommits) [inline math/src/components/Calculator.js:6]: - [ ] **Please make sure that the calculator component follows the provided design guideline.**

Being able to follow a design guideline is a very important skill for any developer. Although your design appears to be exceptional, the motive of providing a design guideline is to make sure that you are well equipped to transform any design into a functional solution. Please make sure that the design of your calculator looks like the original design. 

Origal|Clone
:------:|:-------:
image|image

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @Chike1990, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
## Hi @Chike1990 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 

You have submitted a project, that lacks a professional README file which entails you project description. Kindly refer to this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and make the necessary adjustments.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Chike1990** [comment]
Dear Zawadi,
Thank you for your review.  However, it looks like you didn't check my project well.  If you open the math directory or folder, which was generated by React, you would see that my ReadMe file has been there since my first project this week. Besides, the previous reviewers did not mark this pull request invalid, including the last project for the week. 

Please kindly recheck and let me know what you advise. Thanks for your time

image

File: math/src/components/Calculator.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  width: 300px;
+  text-align: center;
+  border: 1px solid black;
+  border-radius: 5px;
+}
+
+.keys {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: minmax(40px, auto);
```
Comment (Dipeshtwis) [inline math/src/components/Calculator.css]: - [ ] Still, there is some issue with the design. You have added the extra button of `00` which is not required in this design. And also put the default value `0` on the display screen.

### Your design
Screenshot from 2021-09-18 22-02-54


### Given design

Screenshot from 2021-09-18 22-07-13

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Chike1990, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- It's better if you put your README in the root of the branch so that we can easily access the README file instead of going to the `math` folder.
- Also, for future milestones, you can use the `Math-magician` folder as root and put all the files and folders from `math` to the `Math-magician` folder.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chike1990** [comment]
Thanks a lot... I'm going to correct that now

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @Chike1990 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- The project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewers.

 #### Optional suggestions
  - Work on the readme file before you merge it.
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request closed without merging**